build.sbt

```diff
@@ -98,6 +98,7 @@ lazy val service = project
   .enablePlugins(NoPublishPlugin, JavaAppPackaging)
```

Comment:
I can't comment on it but the line above is suspicious to me.
```scala
.dependsOn(sequence % "compile->test", smartgcal % "compile->test")
```
This says that that `Compile` scope of the `service` module depends on the `Test` scope of `sequence` and `smartgcal`. Is that really right? I.e. you want all your test artifacts on your application classpath?

Reply:
I think you're right.  Some additional test dependencies would need to be spelled out but I can update this in a future PR.